tictactoe/test/format_board_tests.erl

```diff
@@ -1,17 +1,20 @@
 -module(format_board_tests).
 -include_lib("eunit/include/eunit.hrl").
 
-get_board_length_test() ->
-  [{ "gets the length of a board",
-     ?assertEqual(9,format_board:get_board_length([1,2,3,4,5,6,7,8,9])),
-     ?assertNotEqual(8,format_board:get_board_length([1,2,3,4,5,6,7,8,9]))}].
+get_board_length_test()		
+	-> [{"gets the length of a board",
+	     ?assertEqual(9,format_board:get_board_length([1,2,3,4,5,6,7,8,9])),
+	     ?assertNotEqual(8,format_board:get_board_length([1,2,3,4,5,6,7,8,9]))}].
 
-get_board_row_length_test() ->
-	[{ "gets the length of a board row",
-		?assertEqual(3,format_board:get_board_row_length(9)),
-		?assertNotEqual(4,format_board:get_board_row_length(9))}].
+get_board_row_length_test()	
+	-> [{"gets the length of a board row",
+	     ?assertEqual(3,format_board:get_board_row_length(9)),
+	     ?assertNotEqual(4,format_board:get_board_row_length(9))}].
```

Comment:
👀

Reply:
r those eyes due to the fact that these redundant test came back again somehow?! AHHHH my bad!